public/index.html

```diff
@@ -0,0 +1,5 @@
+<form action = '/submit'>
+  Name: <input type = 'text' name = 'a1'/>
+  Comment: <input type = 'text' name = 'a2'/>
+  Submit: <button type = 'submit' name = 'b1'> Submit </button>
```

Comment:
what does b1 do? If you don't understand name's functionality please ask me. In this case input would be a shorter code solution but it's not mandatory.